hack/godep-save.sh

```diff
@@ -22,40 +22,65 @@ KUBE_ROOT=$(dirname "${BASH_SOURCE}")/..
 source "${KUBE_ROOT}/hack/lib/init.sh"
 source "${KUBE_ROOT}/hack/lib/util.sh"
 
+GODEP_TMPPFX="_godep_save__tmp."
```

Comment:
The other option here would be:
```
mkdir -p "${KUBE_ROOT}/_tmp/"
GODEP_TMPPFX="${KUBE_ROOT}/_tmp/_godep_save__tmp."
```

This would work too as `/_tmp/` is in .gitignore and isn't rsync'd, but would be in tree so it would be on the same device.

Reply:
I'm not aware of anything that cleans up _tmp.

I think either of them would work, as the two are treated similarly by gitignore and rsync. The only folder that is touched by rsync is _output/dockerized/bin, and it's copied to _output/bin. _output is also used by the build-image and generating the release files.

_tmp on the other hand is used by doc generation, by jenkins, and by the godep-licences script:
https://github.com/kubernetes/kubernetes/blob/44c5182187e6c3d890ff0cd01d9b3ee667a0ebeb/hack/verify-godep-licenses.sh#L31-L41

based on that, _tmp seems like the right candidate. also looks like some cleanup could be done in that verify script to match this one (but that could also be a separate PR)

Reply:
pushed now with _tmp.  We should probably add that to `make clean` though.